trymito.io/components/ExpandableCard/ExpandableCard.module.css

```diff
@@ -0,0 +1,56 @@
+.expandable_card_container {
```

Comment:
Let's add a default border, so you can see it. 

Let's also space the 3 AI sections out a bit better.